public/js/Pull.js

```diff
@@ -100,19 +100,8 @@ define(['underscore', 'appearanceUtils', 'socket'], function(_, utils, socket) {
          return _.findWhere(this.labels, {title: labelName});
       },
 
-      build_status: function() {
-         var status = this.status.commit_status;
-         return status && status.data.state;
-      },
-
-      build_failed: function() {
-         var status = this.build_status();
-         return status === 'failure' || status === 'error';
-      },
-
-      build_succeeded: function() {
-         return this.build_status() === 'success';
-      },
+      build_failed: () => !this.passing,
```

Comment:
Nothing to block on, but in ES6 you can do

```js
build_failed() {
   return this.passing;
}
```

but it's basically the same.

I'm not sure what the convention is for having `this` bound functions as object methods. I guess it only matters if you're planning on using `call` or `apply`, but maybe that's bad practice.

Reply:
Got removed later, I felt like it was silly to have one method that just negated the other.